docs/secure-communication-agents.asciidoc

```diff
@@ -186,14 +254,12 @@ You can specify the following options in the `apm-server.api_key.*` section of t
 +{beatname_lc}.yml+ configuration file.
 They apply to API key communication between the APM Server and APM Agents.
 
-These are different from the API key settings used for the Elasticsearch output and monitoring.
+NOTE: These settings are different from the API key settings used for Elasticsearch output and monitoring.
 
 [float]
 ===== `enabled`
 
 Enable API key authorization by setting `enabled` to `true`.
-Agents will include a valid API key in the following format: `Authorization: ApiKey <token>`.
-The key must be the base64 encoded representation of the API key's `id:name`.
```

Comment:
Do we have this somewhere else? It's an internal detail for our agents, but Jaeger users need to know how to format the "auth_tag" value if they want to use that feature.

Reply:
Good point. We do mention Jaeger steps in https://www.elastic.co/guide/en/apm/server/current/jaeger.html... but I suppose it doesn't hurt to keep this information here. I'm going to move it around though. It feels weird to me to be sitting in the middle of the `enabled` config.